plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/DestructuringTransformation.xtend

```diff
@@ -375,7 +375,7 @@ class DestructuringTransformation extends Transformation {
 	 * Returns a list of statements that are the root statements of the next outer variable environment directly
 	 * containing the given AST node.
 	 */
-	def private static EList<? super Statement> getContainingVariableEnvironmentContent(EObject astNode) {
+	def public static EList<? super Statement> getContainingVariableEnvironmentContent(EObject astNode) {
```

Comment:
`@SuppressWarnings` doesn't work in xtend files. So I could only remove it which I didn't want.